packages/react-table/src/components/TableComposable/examples/ComposableTableExpandable.tsx

```diff
@@ -16,6 +16,7 @@ interface Repository {
   };
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Are there any unused variables here? this seems unnecessary.

Reply:
This is `next-line`, which is `export const ComposableTableExpandable`... the linter complained about this.. i assume because this demo is only being used by an MD file